api/v3/CiviShareTests/Test01.php

```diff
@@ -68,8 +68,9 @@ function civicrm_api3_civi_share_tests_test01(&$params) {
   $change = \Civi\Api4\ShareChange::create(TRUE)
     ->addValue('change_id', 'TODO')
     ->addValue('change_group_id', null)
-    ->addValue('status', \Civi\Api4\ShareChange::STATUS_LOCAL)
+    ->addValue('status', \Civi\Api4\ShareChange::STATUS_PENDING)
     ->addValue('change_type', 'civishare.change.test')
+    ->addValue('status', 'PENDING')
```

Comment:
`status` is set twice, I suppose the second one is superfluous.

Reply:
yes it is